tests/e2e/test_vdbench_workload.py

```diff
@@ -0,0 +1,296 @@
+"""
+Test to exercise VDBench Workload
+"""
+import logging
+import pytest
+import time
+import os
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility.utils import TimeoutSampler, ocsci_log_path
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility import templating
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs.ripsaw import RipSaw
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import E2ETest, workloads
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+# TODO: move this function to pod.py helper functions
+def retrive_files_from_pod(pod_name, localpath, remotepath):
+    """
+    Download a file from pod
+
+    Args:
+        pod_name (str): Name of the pod
+        localpath (str): Local file to download
+        remotepath (str): Target path on the pod
+
+    """
+    cmd = f'cp {pod_name}:{remotepath} {os.path.expanduser(localpath)}'
+    ocp_obj = OCP()
+    ocp_obj.exec_oc_cmd(command=cmd)
+
+
+# TODO: move this function to helper.py helper functions
+def get_labeled_nodes(label):
```

Comment:
@Avilir I see @prsurve also added similar functionality in https://github.com/red-hat-storage/ocs-ci/pull/1345/files#diff-ffa18f30e526ae7b2498579992b406efR272
Let's have 1 implementation for that

Reply:
i will implement it while it will be merged